cli/common/utils_test.go

```diff
@@ -0,0 +1,30 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsValidLabel(t *testing.T) {
+	cases := []struct {
+		name     string
+		label    string
+		expected bool
+	}{
+		{"Valid label", "such-a-good-label", true},
+		{"Invalid label empty", "", false},
+		{"Invalid label contains capital letters", "Peppertrout", false},
+		{"Invalid label contains underscores", "this_is_not_python", false},
+		{"Invalid label too long", "a-very-very-very-long-label-that-is-more-than-63-characters-long", false},
+		{"Invalid label starts with a dash", "-invalid-label", false},
+		{"Invalid label ends with a dash", "invalid-label-", false},
```

Comment:
I like how broke down previous test cases, but it looks like we lost `Standard name with leading numbers works.` test case

Reply:
Agreed, just an oversight from me. Thank you for catching it!